pkg/argocd/git.go

```diff
@@ -60,6 +76,19 @@ func TemplateCommitMessage(tpl *template.Template, appName string, changeList []
 	return cmBuf.String()
 }
 
+// Get repo name and commit id from Image tag and image name
+func GetApplicationDetails(newTag, Image string) (string, string) {
+	applicationName := (strings.Split(Image, "/"))[1]
+	Commit := (strings.Split(newTag, "-"))
+	var CommitId string
+	if len(CommitId) < 5 {
```

Comment:
comment? what's going on here

Reply:
In our case image tags are following two formats checking the format and taking commit Id from image tag.
image formats are ^branch.*-cijob-[0-9]*-commit-commitsha, ^branch-commitsha